src/core/lib/security/credentials/google_default/google_default_credentials.cc

```diff
@@ -314,18 +289,56 @@ grpc_channel_credentials* grpc_google_default_credentials_create() {
     g_metadata_server_available = is_metadata_server_reachable();
   }
   gpr_mu_unlock(&g_state_mu);
+}
+
+static void default_call_creds(
+    grpc_core::RefCountedPtr<grpc_call_credentials>* call_creds,
+    grpc_error* error) {
```

Comment:
I assume this is meant to be an output parameter?  If so, it will need to be `grpc_error**`.

The way this is currently written, you're resetting `error` within the function, but that modified value will never be visible outside of it.

Reply:
Whoops. Totally right. That's embarrassing.